pkg/util/exec/exec.go

```diff
@@ -21,6 +21,9 @@ import (
 	"syscall"
 )
 
+// ErrExecutableNotFound is returned if the executable is not found.
+var ErrExecutableNotFound = osexec.ErrNotFound
```

Comment:
> osexec.ErrNotFound isn't very descriptive

Yeah. In addition this follows this package's general notion of abstracting away os/exec. Currently users of this package don't have to import os/exec at all.